templates/calico-node.service

```diff
@@ -1,45 +1,46 @@
 [Unit]
 Description=calico node
-After=docker.service
-Requires=docker.service
 
 [Service]
 User=root
 Environment=ETCD_ENDPOINTS={{ connection_string }}
 PermissionsStartOnly=true
-ExecStartPre=-/usr/bin/docker rm -f calico-node
-ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node \
-  -e ETCD_ENDPOINTS={{ connection_string }} \
-  -e ETCD_CA_CERT_FILE={{ etcd_ca_path }} \
-  -e ETCD_CERT_FILE={{ etcd_cert_path }} \
-  -e ETCD_KEY_FILE={{ etcd_key_path }} \
-  -e NODENAME={{ nodename }} \
-  -e IP={{ ip }} \
-  -e NO_DEFAULT_POOLS=true \
-  -e AS= \
-  -e CALICO_LIBNETWORK_ENABLED=true \
-  -e IP6= \
-  -e CALICO_NETWORKING_BACKEND=bird \
-  -e CALICO_DISABLE_FILE_LOGGING=true \
-  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
-  -e FELIX_IPV6SUPPORT=false \
-  -e FELIX_LOGSEVERITYSCREEN=info \
-  -e FELIX_HEALTHENABLED=true \
-  -e FELIX_PROMETHEUSREPORTERENABLED=true \
-  -e FELIX_PROMETHEUSREPORTERPORT=9081 \
-  -e FELIX_FLOWLOGSFILEENABLED=true \
-  -e FELIX_FLOWLOGSFILEINCLUDELABELS=true \
-  -e FELIX_FLOWLOGSFILEINCLUDEPOLICIES=true \
-  -e FELIX_FLOWLOGSENABLENETWORKSETS=true \
-  -v /var/run/calico:/var/run/calico \
-  -v /var/lib/calico:/var/lib/calico \
-  -v /lib/modules:/lib/modules \
-  -v /run/docker/plugins:/run/docker/plugins \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -v /var/log/calico:/var/log/calico \
-  -v /opt/calicoctl:/opt/calicoctl \
+ExecStartPre=-/usr/local/sbin/charm-env --charm tigera-secure-ee conctl delete calico-node
+ExecStart=/usr/local/sbin/charm-env --charm tigera-secure-ee conctl run \
```

Comment:
Does `charm-env --charm tigera-secure-ee` work if tigera-secure-ee is deployed with a different application name?

Reply:
It looks like the charm-env code does indeed look at charm name, not application name: https://github.com/juju-solutions/layer-basic/blob/1e25cf7d63285f222207efb4b189f8c076b78b2d/bin/charm-env#L25

So this should be fine. :+1:

Reply:
Ah, thanks for checking!